src/modules/ekf2/EKF/baro_bias_estimator.hpp

```diff
@@ -92,15 +91,17 @@ class BaroBiasEstimator
 private:
 	float _state{0.f};
 	float _state_max{100.f};
-	float _state_drift_rate{0.005f}; ///< in m/s
 	float _dt{0.01f};
 
 	float _gate_size{3.f}; ///< Used for innovation filtering (innovation test ratio)
 	float _state_var{0.1f}; ///< Initial state uncertainty variance (m^2)
 	float _process_var{25.0e-6f}; ///< State process noise variance (m^2/s^2)
 	float _state_var_max{2.f}; ///< Used to constrain the state variance (m^2)
 
-	AlphaFilter<float> _signed_innov_test_ratio_lpf; ///< innovation sequence monitoring; used to detect a bias in the state
+	// Innovation sequence monitoring; used to detect a bias in the state
+	AlphaFilter<float> _signed_innov_test_ratio_lpf;
+	float _time_since_last_negative_innov{0.f};
```

Comment:
It likely doesn't matter with these if you're only checking the magnitude later, but in general we really shouldn't be storing potentially unbounded time in single precision floating point.